docs/develop/mcp/launch_mcp_server.md

```diff
@@ -94,6 +104,10 @@ Where:
   - `host`: host mode.
 - `mcp-host-api_key`: Required in self-host mode to authenticate the MCP server with the RAGFlow server. See [here](../acquire_ragflow_api_key.md) for instructions on acquiring an API key.
 
+:::tip INFO
+Host mode need to combined with --no-transport-streamable-http-enabled flag, namely, host+streamable-http is not supported yet.
```

Comment:
```suggestion
If you set `mcp-mode` to `host`, you must add the ·--no-transport-streamable-http-enabled` flag, because the streamable-HTTP transport is not yet supported in host mode.
```